node/node.go

```diff
@@ -290,20 +290,20 @@ func (n *Node) GetKeyValuePairs(address string, options api.AccountQueryOptions,
 			return make(map[string]string), adaptedBlockInfo, nil
 		}
 
-		return nil, api.BlockInfo{}, err
+		return nil, blockInfo, err
 	}
 
 	if check.IfNil(userAccount.DataTrie()) {
-		return map[string]string{}, api.BlockInfo{}, nil
```

Comment:
maybe just here return you need to return the blockinfo, the others with error should still return zero values

Reply:
fixed